### PR TITLE
Reduce trial numbers for PAI IT

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

File: test/config/training_service.yml [hunk 1]
```diff
@@ -78,6 +78,8 @@ paiYarn:
 pai:
   nniManagerIp:
   maxExecDuration: 15m
+  maxTrialNum: 1
```
Comment (QuanluZhang) [inline test/config/training_service.yml:81]: reduced to only 1?

Comment (chicm-ms) [reply test/config/training_service.yml:81]: Please add comment here why change maxTrialNum.

Comment (SparkSnail) [reply test/config/training_service.yml:82]: > reduced to only 1?

1 trial job per example is enough for testing the example case in PAI. This mastrialNum=1 only works for PAI, and other features like NNICTL, assessor etc. could be tested by other pipelines.

Comment (SparkSnail) [reply test/config/training_service.yml:82]: > Please add comment here why change maxTrialNum.

updated.

File: test/config/training_service.yml [hunk 2]
```diff
@@ -78,12 +78,15 @@ paiYarn:
...
+  # PAI has job submission limitation, set maxTrialNum=1 to control trial job numbers for PAI 
+  maxTrialNum: 1
+  trialConcurrency: 1
   paiConfig:
     host:
     userName:
   trainingServicePlatform: pai
   trial:
-    gpuNum: 1
+    gpuNum: 1
```
Comment (chicm-ms) [inline test/config/training_service.yml:89]: it seems a space is added here

**Pull request merged**